db/schema.rb

```diff
@@ -10,9 +10,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_01_07_161848) do
+ActiveRecord::Schema.define(version: 2021_01_25_214324) do
 
   # These are extensions that must be enabled in order to support this database
+  enable_extension "dblink"
```

Comment:
:thinking: this _shouldn't_ be here except in development, but db/schema.rb is really only used for development and testing, so I think it's ok.

Reply:
The assumption of `schema.rb` not being used in production could likely change, esp. when we decide to clean up old migration files. Keeping the extension should not do any harm, I guess.